server/src/main/scala/za/co/absa/atum/server/model/PlayJsonImplicits.scala

```diff
@@ -93,4 +94,7 @@ object PlayJsonImplicits {
   implicit val readsAtumContextDTO: Reads[AtumContextDTO] = Json.reads[AtumContextDTO]
   implicit val writesAtumContextDTO: Writes[AtumContextDTO] = Json.writes[AtumContextDTO]
 
+  implicit def readsSingleSuccessResponse[T: Reads]: Reads[SingleSuccessResponse[T]] = Json.reads[SingleSuccessResponse[T]]
+  implicit def writesSingleSuccessResponse[T: Writes]: Writes[SingleSuccessResponse[T]] = Json.writes[SingleSuccessResponse[T]]
```

Comment:
I think that there is almost nothing going on in these 2 functions. Thus, `Reads` and `Writes` can be possibly combined into 1, i.e. using `Format`. Possibly something like:

```scala
implicit def formatSingleSuccessResponse[T: Format]: Format[SingleSuccessResponse[T]] =
   Json.format[SingleSuccessResponse[T]]
```

I haven't tried it. What do you think? Perhaps we can consolidate some Reads+Writes into Format here in this module

Reply:
yeah seems that it's ok, compiles without any issues ... changed it for the envelopes' types ... for the rest well I would say we could maybe finally replace play-json with circe ...

Reply:
Okay, I'm fine with either doing that now (either of these two :D up to you)  - please can you check if there is a ticket for circe replacement and if not then create one?

Reply:
awesome, thanks!